src/js/vre/chapter36/config/form.js

```diff
@@ -141,27 +151,64 @@ const formConfig = {
             }
           }
         }
-      },
-      veteranInformation: {
-        title: 'Veteran Information',
-        pages: {
-        }
-      },
-      additionalInformation: {
-        title: 'Additional Information',
-        pages: {
-        }
-      },
-      militaryHistory: {
-        title: 'Military History',
-        pages: {
-        }
-      },
-      contactInformation: {
-        title: 'Contact Information',
-        pages: {
+      }
+    },
+    veteranInformation: {
+      title: 'Veteran Information',
+      pages: {
+        veteranInformation: {
+          title: 'Veteran Inforation',
+          path: 'veteran-information',
+          uiSchema: {
+            veteranFullName: fullNameUI,
+            veteranDateOfBirth: currentOrPastDateUI('Date of birth'),
+            veteranSocialSecurityNumber: ssnUI,
+            veteranVaFileNumber: {
+              'ui:title': 'VA File Number',
+              'ui:errorMessages': {
+                pattern: 'Your VA file number must be between 7 to 9 digits'
+              }
+            },
+            'view:veteranDateOfDeathMIAPOW': {
```

Comment:
What you have will work, but we often do something like this to avoid creating a `view:` object if we don't have to:

```js
veteranDateOfDeathMIAPOW: _.merge(
  currentOrPastDateUI('Date of Veteran’s death or date listed as missing in action or POW'),
  {
    'ui:options': {
      hideIf: (formData) => formData['view:isVeteran'] === true,
    }
  }
),
```